sw/device/lib/crypto/data/crypto_testplan.hjson

```diff
@@ -123,5 +125,17 @@
       bazel: ["//sw/device/tests/crypto:symmetric_keygen_functest"]
       lc_states: ["PROD"]
     }
+    {
+      name: crypto_lib_entropy
+      desc: '''Check the quality of entropy generated by PRNGs.
```

Comment:
I am not sure that a known-answer test is supposed to test the "quality" of the entropy, it is really more testing that the PRNG follows the specification

Reply:
Good point. I rephrased it.